opentech/apply/users/templates/users/password_reset/form.html

```diff
@@ -17,7 +17,7 @@
             </div>
         {% endif %}
 
-        <form class="form form--login" method="post">
+        <form class="form form--with-p-tags" method="post">
             {% csrf_token %}
             <p>{{ form.email.label_tag }}</p>
```

Comment:
label_tag is `<label>` and it is best practice to use them for forms.

Reply:
Agreed, however the form is output with surrounding <p> tags [here](https://github.com/OpenTechFund/opentech.fund/pull/75/files/fa7f48d97b6578b2f1506bb21202bc9fae7604e1#diff-094070c446ed733c0fcc633f442f50bdR18) which are used for styling